theme_vehicle/static/src/scss/primary_variables.scss

```diff
@@ -117,12 +117,9 @@ $o-website-values-palettes: (
         'btn-padding-x': 1.15rem,
         'btn-border-radius': 1px,
         'btn-border-radius-sm': 0,
-        'btn-border-radius-lg': 2px,
+        'btn-border-radius-lg': 1px,
         'btn-padding-y': .474rem,
-        'btn-padding-y-lg': 1rem,
-        'btn-padding-x-lg': 3.3rem,
         'btn-font-size': 1.2rem,
-        'btn-font-size-lg': 2rem,
```

Comment:
Hello sorry @xlu-odoo but I tested and with this, the large buttons are really pretty much the same as the normal one... you can reduce the size if needed* but make it so the option still make some sense (= large is bigger than medium).

\* normally we should not merge those anymore as this will impact website currently using the current style...